src/main/java/org/sonar/plugins/stash/StashPluginUtils.java

```diff
@@ -1,38 +1,48 @@
 package org.sonar.plugins.stash;
 
+import java.math.RoundingMode;
 import java.text.DecimalFormat;
-import java.text.MessageFormat;
-import java.text.ParseException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.sonar.api.issue.Issue;
 
-public class StashPluginUtils {
+public final class StashPluginUtils {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(StashPluginUtils.class);

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "LOGGER" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)